arango/client.py

```diff
@@ -31,6 +31,10 @@ class ArangoClient:
         multiple host URLs are provided). Accepted values are "roundrobin" and
         "random". Any other value defaults to round robin.
     :type host_resolver: str
+    :param resolver_max_tries: Number of attempts to process an HTTP request
+        before throwing a ConnectionAbortedError. Must not be lower than the
+        number of hosts.
```

Comment:
We should add a check for number of hosts < max tries.

Reply:
currently we have this over [here](https://github.com/ArangoDB-Community/python-arango/pull/184/files#diff-f698f829422bf3c3602d6df4a9b8d137054a74c366a6d700aedcf8854c92efc0R18-R19), but I can move it to `client.py` if you'd like?

Reply:
After giving it some thought I think it would be best if we leave it within `resolver.py`

Reply:
After giving it some thought I think it would be best if we left it within `resolver.py`

Reply:
Oh right. Yes I think it's ok to leave it in `resolver.py`.

Reply:
sounds good!